doc/demo-alpha.4/README.md

```diff
@@ -46,16 +46,16 @@ node1_uri=$(lnp1-cli info | awk '/node_id/ {print $2"@127.0.0.1"}')
 ```
 
 
-## docker
+## Docker
 
-In order to create a simple setup that allows to test interactions between lnp-nodes, we use `docker-compose` together with some helper aliases. You will have CLI access to a couple of lnp-nodes that will establish connections and channels between them.
+In order to create a simple setup that allows you to test the interactions between lnp-nodes, we use `docker-compose` together with some helper aliases. You will have CLI access to a couple of lnp-nodes that will establish connections and channels between them.
 
-#### requirements
+#### Requirements
 - [git](https://git-scm.com/downloads)
 - [docker](https://docs.docker.com/get-docker/)
 - [docker-compose](https://docs.docker.com/compose/install/)
 
-### setup
+### Setup
 ```bash=
 git clone https://github.com/LNP-BP/lnp-node
 cd lnp-node/demo
```

Comment:
just noticed that I forgot to update this path after structure rework. Should be `cd lnp-node/doc/dempo-alpha.4`.